IdentityCore/src/oauth2/token/MSIDAccessToken.m

```diff
@@ -170,6 +170,21 @@ - (MSIDCredentialType)credentialType
     return MSIDAccessTokenType;
 }
 
+#pragma mark - RefreshNeeded
+
+-(BOOL)refreshNeeded
+{
+    if(self.cachedAt && [[NSDate date] compare:self.cachedAt] == NSOrderedAscending)
```

Comment:
nit: add a whitespace after if and before "("

Reply:
Done